src/pages/public/HelperRegister.tsx

```diff
@@ -0,0 +1,358 @@
+import { PrimaryButton } from "~/components/Button";
+import { FormTitle } from "~/components/Form";
+import { Checkbox, Radio } from "~/components/Form/components";
+import { CheckIcon } from "~/components/Icon";
+import { Step, StepContent, useSteps } from "~/components/Steps";
+import { Title } from "~/components/Title";
+
+type FormProps = FormikProps<typeof initialValues>;
+type PassedFormProps = Pick<FormProps, "setFieldValue" | "values">;
+
+const checklistTexts = [
+  "Über 18 Jahre sein",
+  "In körperlich guter Verfassung sein"
+];
+
+export const steps: Step[] = [
+  {
+    title: "Hinweis",
+    Icon: Info,
+    okText: "Okay, verstanden"
+  },
+  {
+    title: "Über dich",
+    Icon: User,
+    okText: "Weiter"
+  },
+  {
+    title: "Konditionen",
+    Icon: Briefcase,
+    okText: "Weiter"
+  },
+  {
+    title: "Fertig!",
+    Icon: CheckCircle,
+    okText: "Weiter"
+  }
+];
+
+const contents: (StepContent | StepContent[])[] = [
+  {
+    stepIndex: 0,
+    Content: () => (
+      <>
+        <p className="mb-8">
+          Bevor du fortfährst: Zuverlässigkeit steht an oberster Stelle! Wenn du
+          kurzfristig keine Zeit hast, sag bitte direkt der Kontaktperson ab,
+          damit sie umplanen kann. Weiterhin musst du:
+        </p>
+        <ul className="mb-4">
+          {checklistTexts.map((text, i) => (
+            <li key={i} className="flex align-top pl-8 mb-2">
+              <div className="inline-block -ml-8 mr-2">
+                <CheckIcon />
+              </div>
+              <p className="inline-block font-medium">{text}</p>
+            </li>
+          ))}
+        </ul>
+      </>
+    )
+  },
+
+  {
+    stepIndex: 1,
+    Content: ({ setFieldValue, values }: PassedFormProps) => (
+      <>
+        {[
+          ["account.firstName", "Vorname"],
+          ["account.lastName", "Nachname"],
+          ["account.phone", "Telefonnummer"],
+          ["account.email", "E-Mail"],
+          ["account.password", "Passwort"],
+          ["account.password_confirmation", "Passwort wiederholen"]
+        ].map(([key, label]) => (
+          <TextField
+            key={key}
+            label={label}
+            className="mb-4 text-brand"
+            fullWidth
+            onChange={(e) => setFieldValue(key, e.currentTarget.value)}
+          />
+        ))}
+        <div className="mb-2">
+          <FormTitle as="h2" className="mb-2">
+            Du bist
+          </FormTitle>
+          <div>
+            {[
+              ["Schüler*in", "Schüler*in"],
+              ["Student*in", "Student*in"],
+              ["Arbeitslos", "Arbeitslos"],
+              ["Beschäftigt", "Beschäftigt"]
+            ].map(([label, value], i) => {
+              const checked = value === values.employmentStatus;
+
+              return (
+                <Radio
+                  key={i}
+                  checked={checked}
+                  onClick={() => setFieldValue("employmentStatus", value)}
```

Comment:
``` javascript
Warning: Failed prop type: You provided a `checked` prop to a form field without an `onChange` handler. This will render a read-only field. If the field should be mutable use `defaultChecked`. Otherwise, set either `onChange` or `readOnly`.
    in input (at Radio.tsx:23)
    in label (at Radio.tsx:16)
    in Radio (at HelperRegister.tsx:104)
    in div (at HelperRegister.tsx:94)
    in div (at HelperRegister.tsx:90)
    in Content (at HelperRegister.tsx:319)
    in div (at HelperRegister.tsx:314)
    in Formik (at HelperRegister.tsx:306)
    in div (at HelperRegister.tsx:305)
    in HelperRegister (created by Context.Consumer)
    in Route (at routes/index.tsx:46)
    in Switch (at routes/index.tsx:40)
    in AppRoutes (at src/index.tsx:25)
    in Router (at src/index.tsx:24)
    in ThemeProvider (at src/index.tsx:23)
    in Provider (at src/index.tsx:22)
```

```suggestion
                  checked={checked}
                  onChange={() => setFieldValue("employmentStatus", value)}
```